app/models/upvs/message_template.rb

```diff
@@ -106,6 +106,10 @@ def build_message_from_template(message)
       filled_content.gsub!(template_item[:placeholder], message.metadata['data'][template_item[:name]].to_s)
     end
 
+    message.update(
```

Comment:
Na tejto metode sa mi mierne nepozdava, ze `build` robi vlastne create. Za normalnych okolnosti by to malo byt len vytvorenie do pamate bez ulozenia do db, ale asi tomu nieco brani lebo aj vyssie sa to deje.